irodsinfo/tests/test_permissions.py

```diff
@@ -3,33 +3,48 @@
 from django.urls import reverse
 
 # Projectroles dependency
-from projectroles.tests.test_permissions import TestPermissionBase
+from projectroles.tests.test_permissions import TestProjectPermissionBase
 
 
-class TestIrodsinfoPermissions(TestPermissionBase):
+class TestIrodsinfoPermissions(TestProjectPermissionBase):
```

Comment:
Irodsinfo is a site app, so it doesn't have a project context. Hence these tests should be achievable with `TestPermissionBase`.

Since there is no project context, we also only need to test for:

- Regular user (non-superuser)
- Superuser
- Anonymous

In the future, we should set up a site app permission test base class to SODAR Core to simplify this. I'll add an issue for that.

Meanwhile, you can just init the required users manually in `setUp()` just like the original tests did.